overlord/snapstate/snapstate.go

```diff
@@ -179,6 +179,12 @@ func doInstall(st *state.State, snapst *SnapState, snapsup *SnapSetup, flags int
 		prev = setupSecurityPhase2
 	}
 
+	// auto-connections
+	autoConnect := st.NewTask("auto-connect", fmt.Sprintf(i18n.G("Automatically connect eligible plugs and slots of snap %q"), snapsup.Name()))
+	addTask(autoConnect)
+	autoConnect.JoinLane(st.NewLane())
```

Comment:
I missed this, why a new lane here?

Reply:
That doesn't make sense, I'm not sure why did I add this. Removed.